### PR TITLE
Update books - added publication dates.

### PR DESCRIPTION
**Pull request by GemmaG**
Long time watcher, first time user :) This is my first pull request - please let me know if there are any problems! I'm new to Git, so any handy tips are appreciated :)

### CODE REVIEW

File: site/learn/books.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ the front of the book.
 ###  Unix System Programming in OCaml
 <img src="/img/default.png" width="180" style="float: left; margin-right: 15px; margin-bottom: 15px;"></img>
 
-*Xavier Leroy and Didier Rémy*
+*Xavier Leroy and Didier Rémy. May 2010*
```
Comment (avsm) [inline site/learn/books.md]: This line is missing a  "published" beside the date.

**Comment by avsm** [comment]
Thanks for the contribution, Gemma!  There's one minor typo in your changeset -- there is a missing "published" in one of the lines.  If you push an additional change with that fix to this branch, it should be good to merge.

File: site/learn/books.md [hunk 2]
```diff
@@ -8,7 +8,7 @@
 <img src="/img/colour-icon-170x148.png" style="float: left; margin-right: 15px; margin-bottom: 15px;"></img>
 
 *Damien Doligez, Alain Frisch, Jacques Garrigue, Didier Rémy, and Jérôme
-Vouillon*
+Vouillon. Published in 2014*
```
Comment (avsm) [inline site/learn/books.md]: It's probably best to not have a publication date beside the official manual, as it is updated for every release, which will happen every six months.

**Comment by GemmaG** [comment]
Yay :) Thanks - will change.

**Comment by avsm** [comment]
Thanks! This looks fine to me.  I'll let one of the ocaml.org maintainers merge it.

**Comment by pw374** [comment]
Thanks!

**Pull request merged**